test/models/extract_text_from_paper_job_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
+
+class ExtractTextFromPaperJobTest < ActiveSupport::TestCase
+  test 'keep hyphen at wordwrap if it is a name' do
+    text = "Beate Walter-\n" +
+           "Rosenheimer"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.